.github/workflows/ci.yml

```diff
@@ -96,8 +96,8 @@ jobs:
           ls dist
 
       - name: Install from source package
-        run:
-          pip install --pre dist/hdf5plugin*
+        run: |
+          pip install --pre "$(ls dist/hdf5plugin-*)[test]" --only-binary blosc2 || pip install --pre dist/hdf5plugin-*
```

Comment:
blosc2 wheels are not available for python<=3.8, this tries to install blosc2 and retry without it if failed